README.md

```diff
@@ -133,9 +133,46 @@ if __name__ == "__main__":
 
 ![Running a ZenML pipeline](/docs/book/.gitbook/assets/readme_basic_pipeline.gif)
 
-### Deploy workloads easily on your production infrastructure
+### Provision a MLOps stack or reuse existing infrastructure easily
 
-The framework is a gentle entry point for practitioners to build complex ML pipelines with little knowledge required of the underlying infrastructure complexity. ZenML pipelines can be run on AWS, GCP, Azure, Airflow, Kubeflow and even on Kubernetes without having to change any code or know underlying internals.
+The framework is a gentle entry point for practitioners to build complex ML pipelines with little knowledge required of the underlying infrastructure complexity. ZenML pipelines can be run on AWS, GCP, Azure, Airflow, Kubeflow and even on Kubernetes without having to change any code or know underlying internals. 
+
+There are public templates to easily provision a simple MLOps stack that can be used with ZenML:
+
+<div align="center">
+
+[![Create a AWS MLOps Stack](https://img.shields.io/badge/AWS-%23FF9900.svg?style=for-the-badge&logo=amazon-aws&logoColor=white)](https://console.aws.amazon.com/cloudformation/home?region=eu-central-1#/stacks/create/review?stackName=zenml-stack&templateURL=https://zenml-cf-templates.s3.eu-central-1.amazonaws.com/aws-ecr-s3-sagemaker.yaml)
```

Comment:
What was the idea here? What do we want to display or link here?

Reply:
Including these URLs in our documentation doesn't make any sense IMHO for the following reasons:

1. you need a ZenML server deployed remotely, otherwise the stack won't work
2. you have to manually register the ZenML stack. That means copying the JSON ZenML stack representation from the Cloud Formation stack output, using `zenml stack import` to register it (not yet supported).
3. more importantly, you have all of these facilitated for you through the CLI or dashboard

Given the above, I see no reason why we should even publish these URLs outside of the CLI/dashboard interaction.

Reply:
I agree with the comments above, thus resolving this issue.